certified-connectors/SmartCOMM On Demand Doc-Gen/apiDefinition.swagger.json

```diff
@@ -57,7 +58,8 @@
             "type": "string",
             "default": "application/json",
             "description": "The API accepts both XML and JSON responses. The response type is determined by the \u2018Accept\u2019 HTTP header in the request.",
```

Comment:
@tajujsag unicode symbols are not allowed. Can you please look through the file and remove them?